include/FthShortcutState.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <vector>
+#include <algorithm>
+# include "Wlroots.hpp"
+
+#include <iostream>
+#include <functional>
+
+#define FTH_LIMIT_KEY_PRESSED 32
+
+struct binding {
+  std::string name;
+  std::function<void()> action;
+};
+
+struct FthShortcutState {
```

Comment:
Unclear name. I don't know what Fth is supposed to stand for. Also maybe consider using a namespace ?

Reply:
It means Fth

Reply:
I'd remove the prefix then. I don't think brings any information or readability. If you need it to distinguish it from something which is in xkb, put it in a namespace instead.